libraries/AP_RCProtocol/AP_RCProtocol.cpp

```diff
@@ -52,8 +52,10 @@ void AP_RCProtocol::init()
     backend[AP_RCProtocol::CRSF] = new AP_RCProtocol_CRSF(*this);
     backend[AP_RCProtocol::FPORT2] = new AP_RCProtocol_FPort2(*this, true);
 #endif
+#if !defined(IOMCU_FW) && HAL_ENABLE_DSHOT
```

Comment:
this seems to remove fport and st24 on FMU for some boards

Reply:
This will be removed once the rc memory saving PR has gone in